db/schema.rb

```diff
@@ -30,10 +30,12 @@
   end
 
   create_table "recipes", force: :cascade do |t|
-    t.string   "name",        null: false
-    t.text     "description", null: false
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.string   "name",            null: false
+    t.text     "ing_description", null: false
+    t.text     "description",     null: false
+    t.string   "img_url",         null: false
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
     t.index ["name"], name: "index_recipes_on_name", unique: true, using: :btree
   end
 
```

Comment:
Schema is a file that's generated by rails, I don't think I should modify it at all.
@gotar?